libs/src/services/discoveryProvider/DiscoveryProvider.ts

```diff
@@ -997,6 +997,17 @@ export class DiscoveryProvider {
     )
   }
 
+  /**
+   * Retrieves an unclaimed ID
+   * @return encoded ID
+   */
+  async getUnclaimedId(
+    type: 'users' | 'playlists' | 'tracks'
```

Comment:
Wait should this is singular not plural?

Reply:
yea cuz it's the API users route which is plural 
https://discoveryprovider.audius.co/v1/users/unclaimed_id